zebra-state/src/sled_state.rs

```diff
@@ -289,6 +289,13 @@ impl FinalizedState {
         }
     }
 
+    pub fn get_hash(&self, height: block::Height) -> Option<block::Hash> {
+        self.hash_by_height
+            .get(&height.0.to_be_bytes())
+            .expect("sled errors aren't handled")
```

Comment:
Question: what does this expect mean?

Should the errors be handled by sled?
Or should the errors never happen?

```suggestion
            .expect("sled errors aren't handled")
```

Reply:
The errors are likely indicative of IO errors, we could handle them in theory but I think that's a level of reliability we shouldn't concern ourselves with, the idea being if their disk starts failing we just panic at the first sign of trouble.

Reply:
```suggestion
            .expect("the sled database does not have errors")
```

Reply:
Let's use the same message as:
https://github.com/ZcashFoundation/zebra/blob/e05103323e95f76e2b8325c9239846cb2c3eaf71/zebra-state/src/service.rs#L92